firmware/src/transceiver.c

```diff
@@ -545,9 +557,12 @@ static inline void PrepareRDMResponse() {
   if (g_timing_settings.rdm_responder_jitter) {
     jitter = Random_PseudoGet() % g_timing_settings.rdm_responder_jitter;
   }
+  // It's important to stop the timer before changing the period, see 14.3.11
```

Comment:
This is done in the tests now, which is a much more robust way than trying to parse the code.